### ISSUE TITLE
Abnormal ferritin concentration

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6616]
**Preferred term label:** Abnormal ferritin concentration

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Abnormality of iron homeostasis HP:0011031

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
@eswietlik  These terms already exist, see https://hpo.jax.org/app/browse/term/HP:0040133

It is not appropriate to add them as Abnormality of iron homeostasis because ferritin also has other roles.
